src/ol_infrastructure/applications/edxapp/k8s_resources.py

```diff
@@ -211,6 +272,22 @@ def create_k8s_resources(
         opensearch_hostname=opensearch_hostname,
     )
 
+    openedx_data_pvc = kubernetes.core.v1.PersistentVolumeClaim(
```

Comment:
Is this to avoid having to deal with sticky sessions in the CMS?

Reply:
Yes